aiodns/__init__.py

```diff
@@ -80,7 +80,7 @@ def nameservers(self) -> Sequence[str]:
 
     @nameservers.setter
     def nameservers(self, value: Sequence[str]) -> None:
-        self._channel.servers = value
+        self._channel.servers = value if isinstance(value, list) else list(value)
```

Comment:
We can revert this after the next mypy release (also blocking https://github.com/aio-libs/aiodns/pull/138#issuecomment-2818331165)